lib/modules/showParent.js

```diff
@@ -160,7 +160,6 @@ function showCommentHover(thing: Thing) {
 	$parents.find('.usertext-edit').remove(); // edit form
 	$parents.find('.RESUserTag').remove(); // tags
 	$parents.find('.voteWeight').remove(); // tags
-	$parents.find('.entry').removeClass('RES-keyNav-activeElement');
```

Comment:
`showParent` will never be triggered on a non-selected thing, so a parent can have this class.